docs/source/getting_started/getting_started.ipynb

```diff
@@ -45,19 +46,151 @@
+    "![Loading Basic Stable](_static/createws2.png)\n",
+    "\n",
+    "\n",
+    "3. Once the workspace has loaded, you should see a  Jupyter interface that looks like this (note: You will see fewer environments and items in your root directory — this is normal! You may also see some notifications in the bottom right that look like errors about SSH Keys and other things; that is normal as well. You will also see one asking if you would like to take a guided tour.).\n",
+    "![New Workspace View](_static/createws3.png)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Jupyter Interface overview\n",
+    "When you first log in you may see a notification in the bottom right about a guided tour. Feel free to view the tour, which will give you a quick overview of the Jupyter user interface. You can also find the MAAP Tour in the Help menu at any time.\n",
+    "![MAAP Tour Notification](_static/maap_tour_notification.png)\n",
+    "\n",
+    "\n",
+    "In addition to [typical JupyterLab menu bar and sidebar](https://jupyterlab.readthedocs.io/en/stable/user/interface.html) configuration:\n",
+    "\n",
+    "#### MAAP Jupyter Menus\n",
+    "![Jupyter Menus](_static/interface1.png)\n",
+    "\n",
+    " - Data Search: Access Earthdata Search, copy search queries/results\n",
+    " - Git: Open repo in terminal, init, or clone repo\n",
+    " - Help: The help menu has several customized extensions and references to the MAAP documentation.\n",
+    "\n",
+    "#### MAAP Jupyter Sidebar\n",
+    "![Jupyter Sidebar](_static/interface2.png)\n",
+    "\n",
+    " - File Browser\n",
+    " - Running Terminals & Kernels\n",
+    " - Git Repo Interface (if this folder is a Git repo)\n",
+    " - Jupyter Command Search\n",
+    " - Job listing\n",
+    " - Property Inspector\n",
+    " - Open Tabs\n",
+    " - Extension manager\n",
+    "\n",
+    "#### MAAP Blue Sidebar\n",
+    "\n",
+    " - Workspaces: See workspaces, share them, as well as configure settings\n",
+    " - Stacks: See available platforms for workspaces & required memory\n",
+    " - Administration: Control the configuration & policies for your installation.\n",
+    " - Organizations: allow groups of developers to collaborate with private & shared workspaces. Resources & permissions are controlled & allocated within the organization by admin.\n",
+    " - Profile (bottom, labeled with your name): See account info, logout\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### MAAP Storage Options\n",
+    "![Jupyter annotated folders](_static/storageoptions.png)\n",
+    "\n",
+    "#### My root folder (\"in\" the workspace)\n",
+    " - The Jupyter root directory is mounted to `/projects`.  Stuff in here persists across sessions.\n",
```

Comment:
This isn't referred to as the "home" directory here, but in the creating ssh keys section, you refer to the home directory. Should we call this the home directory here to keep the naming more consistent?